frontend/projects/ui/src/app/pages/marketplace-routes/marketplace-show/marketplace-show.module.ts

```diff
@@ -3,9 +3,15 @@ import { CommonModule } from '@angular/common'
 import { Routes, RouterModule } from '@angular/router'
 import { IonicModule } from '@ionic/angular'
 import { MarketplaceShowPage } from './marketplace-show.page'
-import { SharingModule } from 'src/app/modules/sharing.module'
-import { StatusComponentModule } from 'src/app/components/status/status.component.module'
+import {
+  SharedPipesModule,
+  EmverPipesModule,
+  MarkdownPipeModule,
+  TextSpinnerComponentModule,
+  StatusComponentModule,
```

Comment:
We're not actually using this, can delete and remove status component from shared lib.